src/components/DropdownButton/index.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import DropdownButton from '.';
+
+describe('DropdownButton component', () => {
+  it('renders correctly', () => {
+    render(<DropdownButton>Dropdown</DropdownButton>);
+    const child = screen.getByText('Dropdown');
+    expect(child).toBeDefined();
```

Comment:
[should] I'm curious if we want to use this expectation? 

https://jestjs.io/docs/expect#tobedefined
> Use .toBeDefined to check that a variable is not undefined.

What do you think about using this assertion? 
https://github.com/testing-library/jest-dom/#tobeinthedocument

I think it's more clear to someone casually glancing at this, what we're testing for if we use that assertion. We are currently using `toBeTruthy` checks but I think either just keeping that pattern, or using this above linked assertion will be better long term. What do you think?

Reply:
In my opinion, both are the same, if we already have all with `toBeTruthy` we should keep that pattern.

Normally I don't use this kind of assertions, I aim to do things like, get the text of the element and the assert against that, in this case I wanted to do `expect(child.text).toEqual("Dropdown");` but I couldn't easily get the text, and I figured I was focusing too much on something that isn't as important *right now*.

I will change `toBeDefined` to `toBeTruthy` to complete this merge.